### PR TITLE
[GSoD] HVM Troubleshooting

### PR DESCRIPTION
**Pull request by PROTechThor**

### CODE REVIEW

File: user/troubleshooting/hvm-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    grub2-mkconfig --output=/boot/grub2/grub.cfg
+    ~~~
+
+The HVM should now start normally.
+
+## Can't start an OS in an HVM / "Probing EDD (edd=off to disable!... ok" message ##
+
+If you see a screen popup with SeaBios and 4 lines, last one being `Probing EDD (edd=off to disable!... ok`, then enter the following command from a `dom0` prompt:
+
+    qvm-prefs <HVMname> kernel ""
```
Comment (marmarek) [inline user/troubleshooting/hvm-troubleshooting.md:51]: Can you link to a place where this is from? This sounds like a solution for a slightly different issues, so some more context may be needed.

File: user/troubleshooting/hvm-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## HVM crashes when booting from ISO ##
+If your HVM crashes when trying to boot an ISO, first ensure that ` qvm-prefs <HVMname>` is empty, as shown above.
```
Comment (marmarek) [inline user/troubleshooting/hvm-troubleshooting.md:54]: You missed "kernel".

**Review by marmarek — CHANGES_REQUESTED** [review]
Similar to the previous one - changes in the index will conflict, as you included parts of the other PR here.

File: user/troubleshooting/hvm-troubleshooting.md (hunk 1 above)
Comment (PROTechThor) [reply user/troubleshooting/hvm-troubleshooting.md:51]: I got it from the FAQs: https://www.qubes-os.org/faq/#my-hvm-in-qubes-r40-wont-let-me-startinstall-an-os
It was also discussed here: https://groups.google.com/g/qubes-users/c/v_V6c_eW8U8

**Review by marmarek — APPROVED** [review]

**Comment by PROTechThor** [comment]
@marmarek I think I should reference the "[Attached devices in Windows HVM stop working on suspend/resume](https://github.com/QubesOS/qubes-doc/pull/1053/commits/01ba1a27d7f93872c7871a01e99007979a4a9904)" part here as well, don't you think?

**Comment by marmarek** [comment]
Makes sense to add a link, yes.

**Comment by PROTechThor** [comment]
@andrewdavidwong No conflicts here

**Pull request merged**